client/app/components/sections/ManageAvailability/ManageAvailability.js

```diff
@@ -37,14 +39,23 @@ const shouldLoad = (isOpen, prevIsOpen) => isOpen && !prevIsOpen;
    component `isOpen` becomes `true`
 */
 const loadInitialDataIfNeeded = (props, prevProps = null) => {
-  const isOpen = props.winder.isOpen;
-  const prevIsOpen = prevProps && prevProps.winder.isOpen;
+  const isOpen = props.isOpen;
+  const prevIsOpen = prevProps && prevProps.isOpen;
 
   if (shouldLoad(isOpen, prevIsOpen)) {
     props.calendar.onMonthChanged(props.calendar.initialMonth);
   }
 };
 
+const setPushState = (state, title, path) => {
```

Comment:
I'd go with React-Router or similar HoC solution if we need more advanced route manipulation in the future.
Store subscribe is a valid alternative as a short term solution.